qml/PolarGrid.qml

```diff
@@ -25,12 +25,11 @@ Item {
         function formatDistance(distance) {
             // Change precision based in distance
             var precision = 0
-            if(2 < root.maxDistance && root.maxDistance < 5) {
+            if(root.maxDistance < 5) {
                 precision = 1
-            } else if(root.maxDistance < 2) {
-                precision = 2
             }
-            return (distance*scalar).toFixed(precision) + units
+            // Round number to the displayed precision
+            return (Math.round(distance*scalar*Math.pow(10, precision))/Math.pow(10, precision)).toFixed(precision) + units
```

Comment:
cache Math.pow(10, precision):
var precision = Math.pow(10, precision);
return Math.round(distance * scalar * precision / precision ).toFixed(precision) + units.
it's faster and easier to read